Terminal.Gui/Core/PosDim.cs

```diff
@@ -388,7 +391,8 @@ public bool IsFromRemaining ()
 
 			public override string ToString ()
 			{
-				return $"Dim.Factor(factor={factor}, remaining={remaining})";
+				var decChar = CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator;
```

Comment:
Sorry, this isn't right either. We can't have ToString be hard-coded to ignore user culture info. I know this works, but it's not right.

There has got to be a way to make the TESTS set the culture for the test run, so this code is not necessary. 

Like this: https://stackoverflow.com/questions/13354211/how-to-set-default-culture-info-for-entire-c-sharp-application

Reply:
Thank you. That's what I wanted, although the ideal would be for the whole project.